content/posts/enable-beta-addons.md

```diff
@@ -0,0 +1,146 @@
+This allows readers to **find the exact version they are looking for** and also read the documention on their own language if it's available.
+Besides, it contains useful links to go to the project's home, builds page and offline formats.
+
+### Non-stable notification
+
+![Non-stable notification](/images/addons-non-stable-notification.png)
+
+A notification on all _non-stable_ version is shown to clearly communicate readers
+they may be reading an outdated version of the documentation.
+This helps readers to avoid confusions and quickly jump into the _latest_ version of the documentation.
+Besides, it **reduces support requests from old version of your product**.
+
+
+### Latest version notification
+
+![Latest notification](/images/addons-latest-notification.png)
+
+A notification shown on _latest_ version to tell readers
+they are reading the _latest/development_ version of the documentation that may include features not yet deployed.
+This will help your readers to avoid confusions on the version they are using and the one they are reading,
+while letting your project to expose these yet unrelease features.
+
+### Search as you type
+
+![Search](/images/addons-search.gif)
+
+Search _as-you-type_ powered by an Elastic backend that allows users to quickly find exactly what they are looking for while typing.
+It also **saves recent searches to have them at the tip of your fingers**.
+
+
+### Sponsorship
+
+<!--
+
+TODO: not sure if it's worth to explain this here or not.
+Maybe we can have a paragraph similar to the first one in
+https://docs.readthedocs.io/en/stable/advertising/ethical-advertising.html
+and link to that page.
+
+-->
+
+![EthicalAds](/images/addons-ethical-ads.png)
+
+EthicalAds is the business model that makes Read the Docs to continue being a free web service.
+It respects users while providing value to advertisers.
+We don’t track you, sell your data, or anything else.
+We simply show ads to users, based on the content of the pages you look at.
+
+
+## How to enable addons in your project
+
+To enable Read the Docs Addons in your project follow these steps:
+
+1. Go to the [new beta dashboard](https://beta.readthedocs.org).
+1. Click on the project you want to enable addons.
+1. Go to **Settings**, then **Addons (Beta)** from the left bar.
+1. Check **Enable Addons**.
+
+Now you have Read the Docs Addons enabled,
+user of your documentation can start using these features.
+
+## Moving forward
+
+We have implemented Read the Docs Addons in a modular way that allows us to expand them in the future.
+There are plans to keep adding more addons to this list and also re-implement some of the Sphinx extension we've created in the past
+(e.g. [`sphinx-hoverxref`](https://github.com/readthedocs/sphinx-hoverxref)) as an addon,
+making it compatible with non-Sphinx projects.
+
+We are also open to suggestions for new addons, let's us know if you have an idea!
```

Comment:
```suggestion
We are also open to suggestions for new addons, let us know if you have an idea!
```

There's a typo here but this should guide folks based on how you want the feedback. You do this below, but I'd do it here too. If you want Issues in GitHub, link "let us know" to that. If you want an email, link to the contact form.